docs/tutorials/multimodal/object_detection/detection_eval_ddetr_coco.md

```diff
@@ -0,0 +1,92 @@
+
+```python
+checkpoint_name = "deformable_detr_twostage_refine_r50_16x2_50e_coco"
+num_gpus = -1  # use all GPUs
+```
+
+We create the MultiModalPredictor with selected checkpoint name and number of GPUs.
+We also need to specify the problem_type is `"object_detection"`.
+
+```python
+predictor = MultiModalPredictor(
+    hyperparameters={
+        "model.mmdet_image.checkpoint_name": checkpoint_name,
+        "env.num_gpus": num_gpus,
+    },
+    problem_type="object_detection",
+)
+```
+
+Here we use COCO17 to test. 
+See other tutorials for \[Prepare COCO2017], \[Convert VOC Format Dataset to COCO Format], and \[Create Custom Dataset].
+While using COCO dataset, the input is the json annotation file of the dataset split.
+In this example, `instances_val2017.json` is the annotation file of validation split of COCO17 dataset.
+
+```python
+test_path = "coco17/annotations/instances_val2017.json"
+```
+
+To evaluate the pretrained Deformable DETR model we loaded, run:
+
+```python
+predictor.evaluate(test_path)
+```
+
+And the evaluation results is shown in command line output. The first value `0.463` is mAP in COCO standard, and the second one `0.659` is mAP in VOC standard (or mAP50). For more details about these metrics, see [COCO's evaluation guideline](https://cocodataset.org/#detection-eval).
+
+```
+ Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.463
+ Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.659
+ Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.500
+ Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.298
+ Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.493
+ Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.607
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.358
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.603
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.652
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.451
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.692
+ Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.830
+time usage: 389.92
+```
+
+Deformable DETR has best performance but takes more time and (GPU memory) space. 
+If there is a restriction in time or space, 
+see :ref:`sec_automm_detection_eval_fasterrcnn_coco` or :ref:`sec_automm_detection_eval_yolov3_coco`.
+You can also see other tutorials for \[Fast Finetune on COCO format data] or \[Inference on COCO format data (with Visualization)].
+
+### Other Examples
+
+You may go to [AutoMM Examples](https://github.com/awslabs/autogluon/tree/master/examples/automm) to explore other examples about AutoMM.
+
+### Customization
+To learn how to customize AutoMM, please refer to :ref:`sec_automm_customization`.
+
+### Citation
+```
+@inproceedings{
```

Comment:
Have you referred to this citation in the content? If not, using link should be fine.

Reply:
Here I followed MMDet's practice by citing in bibtex format. I could also do an APA/IEEE style if that's prefered.